ramls/circulation.raml

```diff
@@ -568,3 +569,12 @@ resourceTypes:
             exampleCollection: !include examples/request-queue.json
             schemaCollection: requests
         get:
+  /requests-report:
+    displayName: Expired Holds Report
+    /expired-holds:
+      /{servicePointId}:
+        type:
+          collection-get:
+            exampleCollection: !include examples/expired-holds-report.json
+            schemaCollection: report
+        get:
```

Comment:
get declaration is already presented in `collection-get`

Reply:
removed `get` declaration